app/assets/stylesheets/home/layout.sass

```diff
@@ -218,7 +218,7 @@ body
 
 .sponsor__wrapper
   margin: 0 auto
-  max-width: 800px
+  max-width: 1000px
```

Comment:
Why this change?

Reply:
I see, just seems like they're already the biggest thing on the page

<img width="1995" alt="screen shot 2019-02-04 at 11 56 06 am" src="https://user-images.githubusercontent.com/1441807/52223552-dfa51980-2873-11e9-9e00-0f655db0bb32.png">

Reply:
It's also because of the fact that we moved Verizon on the same level as Constellation, GCP, Wegmans